File: code6.cpp
#include <iostream>
#include <vector>
using namespace std;

void UnionArray(vector<int> arr1, vector<int> arr2){
  vector<int> unionArr;
  for(int i = 0 ; i<arr1.size(); i++){
    unionArr.push_back(arr1[i]);
  }
  for(int i = 0 ; i<arr2.size(); i++){
    if(find(unionArr.begin(), unionArr.end(), arr2[i]) == unionArr.end()){
      unionArr.push_back(arr2[i]);
    }
  }
  cout << "Union of the two arrays: ";
  for(int i = 0 ; i<unionArr.size(); i++){
    cout << unionArr[i] << " ";
  }
  cout << endl;
}

int main(){
  vector<int> arr1 = {1, 2, 3, 9, 10};
  vector<int> arr2 = {4, 5, 6};
  UnionArray(arr1, arr2);
  return 0;
}

File: code6.java
import java.util.ArrayList;
import java.util.Arrays;

public class code6 {
  public static void main(String[] args) {

    ArrayList<Integer> arr1 = new ArrayList<>(
        Arrays.asList(1, 2, 3, 4, 5)
    );

    ArrayList<Integer> arr2 = new ArrayList<>(
        Arrays.asList(1, 2, 3, 6, 7, 8, 9, 10)
    );

    ArrayList<Integer> arr3 = new ArrayList<>();

    for (int i = 0; i < arr1.size(); i++) {
      if (arr2.contains(arr1.get(i))) {
        arr3.add(arr1.get(i));
      }
    }

    for (int i : arr3) {
      System.out.println(i);
    }
  }
}
